README.md

```diff
@@ -1,6 +1,6 @@
 # Conche
 
-[![Build Status](http://img.shields.io/travis/kylef/Conche/master.svg?style=flat)](https://travis-ci.org/kylef/Conche)
+[![Build Status](https://img.shields.io/travis/kylef/Conche/master.svg?style=flat)](https://travis-ci.org/kylef/Conche)
```

Comment:
@dkhamsing This bot is super cool.

Seems to be a bug here though, it updated the below links but didn't update the username `kylef` to `Conche` in the above link.